data-dashboard/pages/boards/[id].tsx

```diff
@@ -12,9 +14,9 @@ import ChartTypeMenu, {
   ChartType,
 } from '@/lib/components/ChartTypeMenu/ChartTypeMenu';
 import { DataSourcesMenuModal } from '@/lib/components';
-import { useRouter } from 'next/router';
+import { Chart as ChartModel } from '@/server/models';
+import TemporalDataSourcesListModal from '@/lib/components/TemporalDataSourcesListModal/TemporalDataSourcesListModal';
```

Comment:
I think we should add this component to `@/lib/components/index.ts` so that we can more easily import it. What do you think?

Reply:
I think it doesn't matter, since we are going to remove it anyways when we have the chart flow completed